apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/event/api/EventController.java

```diff
@@ -1,13 +1,8 @@
 package edu.pw.apsienrollment.event.api;
 
-import edu.pw.apsienrollment.common.api.dto.PageRequestDTO;
 import edu.pw.apsienrollment.common.api.dto.SearchRequestDTO;
 import edu.pw.apsienrollment.event.EventService;
-import edu.pw.apsienrollment.event.api.dto.AllowedToCreateDto;
-import edu.pw.apsienrollment.event.api.dto.EventDto;
-import edu.pw.apsienrollment.event.api.dto.EventRequestDto;
-import edu.pw.apsienrollment.event.api.dto.EventWithMeetingsDto;
-import edu.pw.apsienrollment.event.api.dto.MeetingDto;
+import edu.pw.apsienrollment.event.api.dto.*;
```

Comment:
our static analysis engine detects that your pull request includes a change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.